Clear description of three distinctive paths to install SUMA (#950)
uyuni-project/uyuni-docs: modules/installation/pages/install-server-unified.adoc

This section describes how to install {susemgr} Server from the {sle} installation media.
For this topic we assume that you already have valid organization credentials with {scc} and have obtained a registration code for your {susemgr}.

For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:installation:general-requirements.adoc[].

Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:installation:hardware-requirements.adoc[].

[NOTE]
====
* The recommended way of installing {susemgr} is from the {sle} installation media with the Unified Installer.
* In case of installing {susemgr} in a public cloud where {susemgr} image is available, use that image. 

>     For more information, see xref:quickstart-public-cloud:qs-publiccloud-overview.adoc[].

Reviewer: ```suggestion
  For more information, see xref:quickstart-public-cloud:qs-publiccloud-overview.adoc[].
```

> * In case of installing {susemgr} in a public cloud where {susemgr} image is not available, it is possible to change a {sles}{nbsp}{sles-version}{nbsp}{sp-version} into a {susemgr}{nbsp}{productnumber}.

Reviewer 2: ```suggestion
* In case of installing {susemgr} in a public cloud where a {susemgr} image is not available, it is possible to start from a {sles}{nbsp}{sles-version}{nbsp}{sp-version} and switch the base product to {susemgr}{nbsp}{productnumber}.
```

> * In case of installing {susemgr} in a public cloud where a {susemgr} image is not available, it is possible to start from a {sles}{nbsp}{sles-version}{nbsp}{sp-version} and switch the base product to {susemgr}{nbsp}{productnumber}.

Reviewer 3: Any SLE or only JeOS? I seem to remember install-vm.adoc mentions only JeOS.

Author: I have reused the old sentence as it was written before my changes, when there was no mention of JeOS specifically, and had extended it by reference to cloud. You are correct, JeOS specifically is mentioned in the linked document.

Reviewer 3: ACK. Since we mention "public", we should be fine, as the SLE images there should be controlled by SUSE, so there should not be any issues.

>     For more information, see xref:installation:install-vm.adoc[].

Reviewer: ```suggestion
  For more information, see xref:installation:install-vm.adoc[].
```
